### PR TITLE
Update DocumentTabs component visual styles

### PR DESCRIPTION
**Pull request by Cytancy**
## Description:
Update DocumentTab to reflect new design in https://www.figma.com/file/exXS6SS14Qbfba1wyyNyqSlE/Components?node-id=253%3A11209.

## Sandbox server:
https://core-21.sandbox2.eng-dev.flexport.com/

## Jira:
https://flexport.atlassian.net/browse/LDS-154
https://flexport.atlassian.net/browse/LDS-399
https://flexport.atlassian.net/browse/LDS-395

## Manual Testing Plan:
1. Run Storybook.
2. View DocumentTab and test functionality.
3. Run tests, ensure `DocumentTab` test pass and snapshot tests having to do with `latitude.black` and `IconButton` padding pass.
4. Verify `DocumentTabs` show up correctly in Core. (https://core-21.sandbox2.eng-dev.flexport.com/carrier_bookings)
   * Filter on Client that has bookings (e.g. '10414590 Canada')
   * Select a booking and add a shipping order.
   * Observe new `DocumentTabs`.

## Screenshots:
Before:
document_tabs_old

After:
document_tabs_new

Before:
Screen Shot 2019-10-23 at 11 30 21 AM

After:
Screen Shot 2019-10-23 at 11 12 38 AM

### CODE REVIEW

File: colors.js [hunk 1]
```diff
@@ -14,7 +14,7 @@
 import type {Color} from "./styles/colors";
 
 const latitudeColors = Object.freeze({
-  black: "#14171B",
+  black: "#272C34",
```
Comment (FermiDirak) [inline colors.js]: @randallbhom

File: document/DocumentTabs/DocumentTabs.jsx [hunk 2]
```diff
@@ -96,80 +95,99 @@ function DocumentTabs({
...
     display: "flex",
     justifyContent: "center",
     alignItems: "center",
-    width: ADD_BUTTON_WIDTH,
-    border: "none",
-    outline: "none",
-    padding: "0",
-    margin: "0",
+    margin: "0 4px 0 28px",
+    // Hack: since inverted colors for buttons are not yet supported
```
Comment (FermiDirak) [inline document/DocumentTabs/DocumentTabs.jsx]: @randallbhom Thoughts on giving Buttons an inverted prop for the cases where non-solid buttons are needed on dark backgrounds?

File: button/IconButton.jsx [hunk 3]
```diff
@@ -254,4 +255,18 @@ const styles = StyleSheet.create({
       fill: colors.blackDoNotUse,
     },
   },
+  /*
+    Padding differences for IconButton to make icon-only IconButtons perfect
+    squares. deprecatedWhitespace.js and whitespace.js do not support this
+    specific use case.
```
Comment (FermiDirak) [inline button/IconButton.jsx]: 👍

File: document/DocumentTabs/DocumentTabs.jsx (hunk 2 above)
Comment (randallbhom) [reply document/DocumentTabs/DocumentTabs.jsx]: I think it makes sense. @notandrewkaye what are your thoughts?

File: colors.js (hunk 1 above)
Comment (Cytancy) [reply colors.js]: Yep, made this ticket/change for Randall

File: document/DocumentTabs/DocumentTabs.jsx (hunk 2 above)
Comment (Cytancy) [reply document/DocumentTabs/DocumentTabs.jsx]: Should it be in `intent`? I'll create a separate ticket for this.

**Comment by Cytancy** [comment]
@FermiDirak Fixed! Thanks!
Screen Shot 2019-10-23 at 3 26 39 PM

**Pull request merged**